components/create_post/create_post.jsx

```diff
@@ -325,6 +335,10 @@ class CreatePost extends React.PureComponent {
             this.focusTextbox();
         }
 
+        if (this.props.currentChannel.id !== prevState.currentChannel.id) {
```

Comment:
This is total nitpicking, but would you mind changing this to look at `prevProps` instead of `prevState`? I know they should be the same value, but these checks are usually comparing either props to props or state to state instead of mixing the two.
```suggestion
        if (this.props.currentChannel.id !== prevProps.currentChannel.id) {
```